cmd/ipsw/cmd/appstore/appstore_review_ls.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright © 2024 Kenneth H. Cox
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+package appstore
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/apex/log"
+	"github.com/blacktop/ipsw/pkg/appstore"
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+
+	"github.com/spf13/viper"
+)
+
+func init() {
+	ASReviewCmd.AddCommand(ASReviewListCmd)
+
+	ASReviewListCmd.Flags().String("id", "", "App ID")
+	ASReviewListCmd.Flags().String("after", "", "Only show responses on or after date, e.g. \"2024-12-22\"")
+	ASReviewListCmd.Flags().String("since", "", "Only show responses within duration, e.g. \"36h\"")
+	viper.BindPFlag("appstore.review.ls.id", ASReviewListCmd.Flags().Lookup("id"))
+	viper.BindPFlag("appstore.review.ls.after", ASReviewListCmd.Flags().Lookup("after"))
+	viper.BindPFlag("appstore.review.ls.since", ASReviewListCmd.Flags().Lookup("since"))
+}
+
+// ASReviewListCmd represents the appstore review ls command
+var ASReviewListCmd = &cobra.Command{
+	Use:           "ls",
```

Comment:
if there is only 1 sub-cmd, there there shouldn't be a sub-cmd

Reply:
Good point, done